pkgs/development/tools/misc/roswell/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, curl }:
+
+stdenv.mkDerivation rec {
+  name = "${product}-${version}";
+  product = "roswell";
+  version = "17.9.10.82";
+
+  src = fetchFromGitHub {
+    owner = product;
+    repo = product;
+    rev = "v${version}";
+    sha256 = "1064lj5hy70j7nkf46h7pg5yvksgaj3k9aha7gp09hd826857q4m";
+  };
+
+  preConfigure = ''
+    sh bootstrap
+  '';
+
+  nativeBuildInputs = [ autoconf automake curl ];
```

Comment:
What is curl used for here?

Reply:
It's checked for in `configure.ac`, looks like it is more of a runtime requirement. Probably should put it is `buildInputs`, right?

Reply:
Probably, if it is referenced at runtime. Now, it won't actually matter unless build and host platform differ (and are incompatible, in which case curl would crash when invoked on the host machine).

Reply:
I've moved it, but haven't pushed my WIP yet.